charts/ctlog/templates/_helpers.tpl

```diff
@@ -98,10 +98,18 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 Selector labels
 */}}
 {{- define "ctlog.selectorLabels" -}}
```

Comment:
it looks like you are switching the two places that calls `include "ctlog.selectorLabels"` to `include "ctlog.matchLabels"`, we can probably get rid of `selectorLabels` as no one is using that now.